README.md

```diff
@@ -18,28 +18,30 @@ npm install react-remote-resource --save
 yarn add react-remote-resource
 ```
 
-### Simple Example
+### Example
 
 ```jsx
 import {
-  createRemoteResource,
-  useResourceState,
-  RemoteResourceBoundary
+  createResource,
+  createSingleEntryResource,
+  createTimedKeyedResource,
+  useEntryState,
+  RemoteResourceBoundary,
 } from "react-remote-resource";
 
-const userResource = createRemoteResource({
-  load: userId => fetchJson(`/api/users/${userId}`),
-  invalidateAfter: 60 * 60 * 1000 // 1 hour
-});
+const userResource = createSingleEntryResource(
+  userId => fetchJson(`/api/users/${userId}`
+);
 
-const tweetsResource = createRemoteResource({
-  load: userId => fetchJson(`/api/users/${userId}/tweets`),
-  invalidateAfter: 10000 // 10 seconds
-});
+const tweetsResource = createTimedKeyedResource(
+  10000,
+  userId => userId,
+  userId => fetchJson(`/api/users/${userId}/tweets`)
+);
 
 const UserInfo = ({ userId }) => {
-  const [user] = useResourceState(userResource, [userId]);
-  const [tweets] = useResourceState(tweetsResource, [userId]);
+  const [user] = useEntryState(userResource, [userId]);
+  const [tweets] = useEntryState(tweetsResource, [userId]);
```

Comment:
Oh, I like that! I'll change that. I thought changing this to `useEntryState` instead of `useResourceState` made it more obvious that you were going to get back the state for the entry instead of the whole resource state.